esp/scm/ws_esdlconfig.ecm

```diff
@@ -101,6 +101,45 @@ ESPrequest GetESDLBindingRequest
     string EspBindingName;
     string EspPort;       //Name + port = service
     string EsdlBindingId; //espprocname.espbinding
+    [min_ver("1.1")] bool IncludeInterfaceDefinition;
+};
+
+ESPstruct ESDLBinding
+{
+    string EspProcess;
+    string EspBinding;
+    string Id;
+};
+
+ESPstruct ESDLDefinition
+{
+    String Name;
+    int Seq;
+    String Id;
+    [min_ver("1.1")] String Interface;
+};
+
+ESPStruct Attribute
+{
+	String Name;
+	String Value;
+};
+
+ESPStruct MethodConfig
+{
+	String Name;
+	ESParray<ESPStruct Attribute, Attribute> Attributes;
```

Comment:
If you like, you may use:
ESParray<ESPstruct NamedValue, Attribute> Attributes;

The ESPstruct NamedValue is defined in common.ecm